packages/suite/src/reducers/suite/suiteReducer.ts

```diff
@@ -460,16 +466,20 @@ export const selectFirmwareRevisionCheckError = (state: StateForFirmwareChecks)
  */
 const selectIsFirmwareRevisionCheckEnabledAndFailed = (state: StateForFirmwareChecks): boolean => {
     const error = selectFirmwareRevisionCheckError(state);
+    const softErrors: FirmwareRevisionCheckError[] = [
+        'cannot-perform-check-offline',
+        'other-error',
+    ];
 
-    return error ? !['cannot-perform-check-offline', 'other-error'].includes(error) : false;
```

Comment:
I didn't realize, but this was typed as `(string []).includes(FirmwareRevisionCheckError)` so I could write `'other-errooooor'` and TS would be fine, hence `softErrors` declaration.